<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ListOfBorrowedBooks</class>
 <widget class="QDialog" name="ListOfBorrowedBooks">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>861</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>لیست امانت ها</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/img/images/login.png</normaloff>:/img/images/login.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">
QLineEdit{
	border : 1px solid #b0bfae;
	border-radius : 5px;
	background-color : #ebe8e8;
}

QPushButton{
	border-radius: 10px;
	padding : 2px 5px;
	background :#2b58e0;
	border : 2px solid #21698a;
	color : white;
}
QPushButton:hover {
	background-color : #75a0eb ;
}
QToolButton
{
border-radius:10px;
background:red;
color:white;
}


QTableWidget{
selection-background-color: #75a0eb ;
border : 1px solid #ccc;
background-color : #f6f6f6;
alternate-background-color : #e9e9e9;
color : black;
}


</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLineEdit" name="borrowingDate">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>تاریخ امانت گرفتن : </string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLineEdit" name="returnDate">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>تاریخ عودت :</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="tableBooks">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>105</number>
     </attribute>
     <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <column>
      <property name="text">
       <string>آیدی کتاب</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>نام کتاب </string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>نویسنده</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>انتشارات</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>مترجم</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>سال چاپ</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>ژانر</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>رده سنی</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="back">
     <property name="text">
      <string>برگشت</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
